app/src/sharedTest/java/tech/relaycorp/gateway/test/factory/ParcelFactory.kt

```diff
@@ -0,0 +1,30 @@
+package tech.relaycorp.gateway.test.factory
+
+import tech.relaycorp.relaynet.issueGatewayCertificate
+import tech.relaycorp.relaynet.messages.Parcel
+import tech.relaycorp.relaynet.wrappers.generateRSAKeyPair
+import java.time.ZonedDateTime
+
+object ParcelFactory {
+
+    fun build() = Parcel(
+        recipientAddress = "http://example.org",
+        senderCertificate = certificate,
+        payload = "".toByteArray()
+    )
+
+    fun buildSerialized() =
+        build().serialize(keyPair.private)
+
+    private val keyPair by lazy {
+        generateRSAKeyPair()
+    }
+
+    private val certificate by lazy {
+        issueGatewayCertificate(
```

Comment:
I'd suggest to use `issueEndpointGateway` here instead, since parcels are signed by endpoints. Future tests may break if we don't use the right type of certificate.